public/javascripts/SVValidate/src/status/StatusField.js

```diff
@@ -22,13 +22,19 @@ function StatusField() {
     }
 
     /**
-     * Updates the number of labels the user has validated.
-     * @param count {int} Number of labels the user has validated.
+     * Updates the number of labels the user has validated in all time.
+     * @param count {int} Number of labels the user has validated in all time.
      */
     function updateLabelCounts(count) {
         svv.ui.status.labelCount.html(count);
     }
 
+    /**
+     * Increments the number of labels the user has validated by 1
+     */
+    function incrementLabelCounts(){
+        updateLabelCounts(parseInt(svv.ui.status.labelCount.html()) + 1);
```

Comment:
I think it would be better for this file to have a variable that saves the current validation count instead of looking at the HTML to find it. Then `updateLabelCounts` can update that variable.

Reply:
I'm now using the existing svv.validationsCompleted variable to store the total number of validations completed

Reply:
Actually, I think it is best to just never have that global variable. You can pass in `param.completedValidations` to the `StatusField` constructor and save the variable locally in the `StatusField` object.